### PR TITLE
Documentation for docker-sync

### PR DESCRIPTION
**Pull request by torhoehn**
## Description

As mentioned in #290 here's a small start of documentation how to use docker-sync on MacOS X.

### CODE REVIEW

File: Docs/macos_docker_sync/mac.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Development on a Mac using docker-sync
+Due performance issues while developing on a Mac with docker-compose it’s necessary to use docker-sync.
```
Comment (damien-carcel) [inline Docs/macos_docker_sync/mac.md]: Not just docker-compose, Docker.

File: Docs/macos_docker_sync/mac.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Requirements
+Install [docker-sync](http://docker-sync.io/):
+```bash
+gem install docker-sync
```
Comment (damien-carcel) [inline Docs/macos_docker_sync/mac.md]: please add `$ ` before bash commands (or `# ` if they are to be ran as root).

File: Docs/macos_docker_sync/mac.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+```
+
+Install [unison](https://www.cis.upenn.edu/~bcpierce/unison/):
+```bash
+brew install unison
+```
+
+## File locations
+Just like the compose file template copy `docker-sync.yml.dist` and `docker-compose.mac.yml.dist` to the root of your project or next to you `docker-compose.yml` if you placed it somewhere else. In this case, you have to change the `src:` parameter of you configution. Also don't forget to get rid of the `.dist` file extension for both files. ;)
```
Comment (damien-carcel) [inline Docs/macos_docker_sync/mac.md]: configution => configuration 😃

Comment (damien-carcel) [reply Docs/macos_docker_sync/mac.md]: Please precise the file in which you eventually have to change the `src:` parameter.

**Review by damien-carcel — CHANGES_REQUESTED** [review]
A few minor comments.

Just a question: how are to be run the commands that are usually ran with compose? Does that change?

If it does not change and one can then use `docker-compose`, just add a line for it at the end.
If not, please add a small explanation on it.

File: Docs/macos_docker_sync/mac.md (hunk 1 above)
Comment (torhoehn) [reply Docs/macos_docker_sync/mac.md]: Done.

**Review by damien-carcel — APPROVED** [review]

**Pull request merged**